src/lib/components/Start/OnDate.js

```diff
@@ -25,7 +26,19 @@ const StartOnDate = ({
     locale,
     readOnly: true,
   };
-
+  let inputAttributes = {
+    id: `${id}-datetime`,
+    name: 'start.onDate.date',
+    readOnly: true,
+  };
+  if(disabled){
+    inputAttributes = {
+      id: `${id}-datetime`,
+      name: 'start.onDate.date',
+      readOnly: true,    
+      disabled: true,
+    };
+  }
```

Comment:
@DanielIsMe try to not repeat code please:
```
let inputAttributes = {
    id: `${id}-datetime`,
    name: 'start.onDate.date',
    readOnly: true,
};
if (disabled) {
    inputAttributes = {...inputAttributes, disabled }
}
  ```
  or
  ```
...
if (disabled) {
    inputAttributes.disabled = disabled;
}
  ```
  or simpler:
  ```
  const inputAttributes = {
    id: `${id}-datetime`,
    name: 'start.onDate.date',
    readOnly: true,
    disabled
};
```

Reply:
@alexjidras yes, thanks.  i think the first one work, i tried the 2nd and 3rd, if the inputAttributes added disabled , the  date component alwasy disabled